First/lesson_3.py

```diff
@@ -0,0 +1,51 @@
+# 1) Заменить в произвольной строке согласные буквы на гласные.
+
+text = 'bakepimobyn'
+text2 = text.replace('a', 'b')
+text3 = text2.replace('e', 'm')
+text4 = text3.replace('i', 'k')
+text5 = text4.replace('o', 'm')
+text6 = text5.replace('y', 'm')
+print(text6)
```

Comment:
you need to iterate over the string(text) and determine which letter is vowels and which is consonants. You don't need to do it by hand, because if the string(text) will changed, then you will need to rewrite your code